LICENSE

```diff
@@ -1,9 +1,32 @@
-testgres is released under the PostgreSQL License, a liberal Open Source license, similar to the BSD or MIT licenses.
```

Comment:
You should not change this file.

Reply:
Actually, this file seems to be wrong and should be removed. All licenses for packages must be in the package recipe directory.

Reply:
I've just deleted the file

Reply:
@conda-forge/help-python, please review